app/src/main/java/ch/epfl/sdp/blindly/main_screen/my_matches/MyMatchesAdapter.kt

```diff
@@ -41,8 +43,9 @@ class MyMatchesAdapter(
     inner class ViewHolder(view: View) : RecyclerView.ViewHolder(view), View.OnClickListener {
         val matchedName: TextView = view.findViewById(R.id.matchedUserName)
         val userNameLayout: LinearLayout = view.findViewById(R.id.userNameLayout)
-        val expandableChatAndMapLayout: RelativeLayout = view.findViewById(R.id.chatAndMapLayout)
+        val expandableChatAndMapLayout: LinearLayout = view.findViewById(R.id.chatAndMapLayout)
```

Comment:
What was the reason for this switch?

Reply:
Yes, I did it because I added a button and I prefered to deal with an horizontal linear layout. But actually I don't think it changes anything, it is just personal preference. I believe it allows us to write less attributes in the xml file linked to it.